src/idl/wholeBodyDynamics_IDLServer/wholeBodyDynamics_IDLServer.thrift

```diff
@@ -80,6 +80,13 @@ service wholeBodyDynamics_IDLServer
   */
   bool resetOffset(1:string calib_code)
 
+  /**
+   * Use the offline estimated offset of the sensor.
+   * @param calib_code argument to specify the sensors to reset (all,arms,legs,feet)
+   * @return true/false on success/failure
+   */
+  bool usePreEstimatedOffset(1:string calib_code)
```

Comment:
The `calib_code` is a legacy parameter for the other methods. Given that `usePreEstimatedOffset` is a new method, perhaps we can just avoid introducing `calib_code` at all?

Reply:
I had a similar discussion with @prashanthr05, I was thinking of leaving it for the possibility of giving options later on such as selecting in which sensors to use it.  But I think that I will add it when I work for it and for now it might be better to not receive parameters.

Reply:
Done no parameters for now